scripts/live_test/monitor.yml

```diff
@@ -17,4 +17,4 @@ jobs:
     - bash: |
         pip install mysql-connector-python
         pip install requests
-        python /home/vsts/work/1/s/scripts/live_test/monitor.py "$(DB_PWD)" "$(GITHUB_TOKEN)"
\ No newline at end of file
+        python /mnt/vss/_work/1/s/scripts/live_test/monitor.py "$(DB_PWD)" "$(GITHUB_TOKEN)"
```

Comment:
After using the 1ES agent pool, the pipeline working directory has changed, need to change /home/vsts/work to /mnt/vss/_work

Reply:
Is it possible to use [predefined variables](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml) instead of hardcoding it to `/mnt/vss/_work/1/s`?

Reply:
The building DEB pipeline uses `BUILD_SOURCESDIRECTORY` as `/home/vsts/work/1/s`:

https://github.com/Azure/azure-cli/blob/47ef9eb05d15d884f163e1f9db1237b399a3c245/scripts/release/debian/pipeline.sh#L14